net/udpbroadcastrelay/src/opnsense/mvc/app/controllers/OPNsense/UDPBroadcastRelay/Api/ServiceController.php

```diff
@@ -31,7 +31,6 @@
 use OPNsense\Base\ApiMutableModelControllerBase;
 use OPNsense\UDPBroadcastRelay\UDPBroadcastRelay;
 use OPNsense\Core\Backend;
-
```

Comment:
if you can bring back this spacer line also ;)

Reply:
Sorry, took out more than I needed to. 
I have now retired and moved out of London and I am busier than ever,  that's my excuse and I'm sticking to it. :)

Reply:
Living the good life I see. Glad to hear you're doing good! :)